commcare/src/main/resources/webapp/js/controllers.js

```diff
@@ -367,6 +367,22 @@
                 && $scope.configOutdated;
         };
 
+        $scope.syncConfig = function() {
+            blockUI();
```

Comment:
Blocking UI here seems pointless. The time required to send an event is so small, that the UI block is literally a blink.